cypress/features/regression/actionPopover.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component page
+      And I open Action Popover component
+
+  @positive
+  Scenario: Open Action Popover component
+    # When I open Action Popover component
+    Then Action Popover component is visible
```

Comment:
the same as above replace 'component' to 'element'

Reply:
fixed